README.md

```diff
@@ -18,6 +18,7 @@ This extension provides following features:
 * Magic [Nette\Object and Nette\SmartObject](https://doc.nette.org/en/2.4/php-language-enhancements) properties
 * Event listeners through the `on*` properties
 * Defines early terminating method calls for Presenter methods to prevent `Undefined variable` errors
+* Understand the exact array shape coming from `Nette\Utils\Strings::matchAll()` based on pattern
```

Comment:
Please mention match() too

Reply:
Adjusted, is this sufficient?

Reply:
No, mention the separately ;) What if Nette adds another method with this pattern and we don't support it? ;)

Reply:
Done